components/style/properties/properties.mako.rs

```diff
@@ -3471,6 +3482,17 @@ pub enum AliasId {
     % endfor
 }
 
+impl fmt::Debug for AliasId {
+    fn fmt(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+        let name = match *self {
+            % for property in data.all_aliases():
+                AliasId::${property.camel_case} => "${property.camel_case}",
+            % endfor
+        };
+        formatter.write_str(name)
```

Comment:
Is the improvement that there is a single `write_str` final call? Because that may be something worth doing in the derivation of `ToCss`, for types with many unit variants.

Reply:
Maybe that helps too, but I think mostly what jdm wrote in the PR message:

>  The more limited Debug implementations enable sharing the property name strings between more functions

Reply:
Oh, right. Well `#[css(derive_debug)] would definitely help with that too for other impls then.